sdk/search/azure-search/src/main/java/com/azure/search/models/ImageAnalysisSkillLanguage.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/**
+ * Defines values for ImageAnalysisSkillLanguage.
+ */
+public final class ImageAnalysisSkillLanguage extends ExpandableStringEnum<ImageAnalysisSkillLanguage> {
```

Comment:
There are a couple more supported values that will be supported with this as part of the latest Azure Search deployment that is currently rolling out.  Would obviously require we update the swagger with them.  Not a big deal if they don't make it into this release, but just worth mentioning, I had had it on my radar to add them soon and we ended up assigning it as an intro task for Jennifer but I'm not sure what the timeline will be on cutting a new release or not.

Reply:
@careyjmac This is necessary for 2019-05-06 because skill definitions cross api-versions, right?

The good news is that this scenario is backward-compatible from the client perspective because the type is an extensible enum.

Reply:
Yes, it will be supported in any api version that the ImageAnalysisSkill is supported in (which is every api version with grok), by design.  Again not a super huge deal if it doesn't exist in the initial release imo just because its additions to the list of available options.

Reply:
We'll take care of this when we do a pass over the Swagger and regenerate the code. Not blocking for this PR.